Cargo.lock

```diff
@@ -400,18 +412,29 @@ version = "0.12.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "8a9ee70c43aaf417c914396645a0fa852624801b24ebb7ae78fe8272889ac888"
 dependencies = [
- "ahash",
+ "ahash 0.7.6",
+]
+
+[[package]]
+name = "hashbrown"
+version = "0.13.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "33ff8ae62cd3a9102e5637afc8452c55acf3844001bd5374e0b0bd7b6616c038"
+dependencies = [
+ "ahash 0.8.2",
 ]
```

Comment:
Looks like this is pulling in an older version of `hashbrown`

Reply:
This PR actually pulls in the newest version of `hashbrown`.
Both `textwrap` and `unicode-linebreaks` still depend on an old version.
`imara-diff` uses the newest version aswell and I do not want to downgrade `hashbrown` in `imara-diff`, because other crates that depend on it (`gitoxide` for example) already use the newest version (and in the future crates will rather upgrade then downgrade).

It's once again one of those unfortunate ecosystem splits where `unicode-lines` and `textwrap` want to maintain a MSRV of 1.56 but  `hashbrown` had raised it MSRV to 1.61.
It's a bit unfortunate that these ecosystem splits happens but I think depending on the newest version and waiting for other dependencies to bump is the correct way to go.